TestsOutsidePackage/test_score.py

```diff
@@ -54,3 +93,272 @@ def test_run_scoring(test_output_dirs: OutputFolderForTests, is_ensemble: bool)
+    """
+    Test the extraction folder contains expected files, not in a folder.
+
+    :param extraction_folder: Path to extracted zip file.
+    :param expected_filenames: List of expected filenames.
+    """
+    # Get all files/folders in this series
+    extracted_files = list(extraction_folder.glob('**/*'))
+    # Check they are all files (no folders) and in the extraction_folder (not a subdirectory)
+    for extracted_file in extracted_files:
+        assert extracted_file.is_file()
+        relative_path = extracted_file.relative_to(extraction_folder)
+        assert str(relative_path.parent) == '.'
+    extracted_file_names = sorted([extracted_file.name for extracted_file in extracted_files])
+    # Check names are as expected
+    assert extracted_file_names == expected_filenames
+
+
+def assert_zip_file_contents(zip_filename: Path, expected_filenames: List[str],
+                             model_folder: Path) -> None:
+    """
+    Compare the contents of a zip file, with a list of expected filenames.
+
+    :param zip_filename: Path to zip file.
+    :param expected_filenames: List of expected filenames.
+    :param model_folder: Scratch folder.
+    """
+    assert zip_filename.is_file()
+    extraction_folder = model_folder / "temp_zip_extraction"
+    with zipfile.ZipFile(zip_filename, 'r') as zip_file:
+        zip_file.extractall(extraction_folder)
+    _common_test_unpack_zip(extraction_folder, expected_filenames)
+
+
+def test_convert_nifti_to_zipped_dicom_rt(test_output_dirs: OutputFolderForTests) -> None:
+    """
+    Test calling convert_nifti_to_zipped_dicom_rt.
+
+    :param test_output_dirs: Test output directories.
+    """
+    model_folder = test_output_dirs.root_dir / "final"
+    model_folder.mkdir()
+
+    nifti_filename, reference_series_folder = convert_zipped_dicom_to_nifti(HN_DICOM_SERIES_ZIP,
+                                                                            model_folder)
+    model_config = PassThroughModel()
+    result_dst = convert_nifti_to_zipped_dicom_rt(HNSEGMENTATION_FILE, reference_series_folder, model_folder,
+                                                  model_config)
+    assert_zip_file_contents(result_dst, HN_DICOM_RT_ZIPPED, model_folder)
+
+
+def test_score_image_dicom_two_inputs(test_output_dirs: OutputFolderForTests) -> None:
+    """
+    Test that dicom in with more than one input fails.
+
+    :param test_output_dirs: Test output directories.
+    """
+    model_folder = test_output_dirs.root_dir / "final"
+
+    score_pipeline_config = ScorePipelineConfig(
+        data_folder=TEST_DATA_DIR,
+        model_folder=str(model_folder),
+        image_files=[str(HN_DICOM_SERIES_ZIP), str(HN_DICOM_SERIES_ZIP)],
+        result_image_name=HNSEGMENTATION_FILE.name,
+        use_gpu=False,
+        use_dicom=True)
+
+    with pytest.raises(ValueError):
+        score_image(score_pipeline_config)
```

Comment:
please also assert that this is raising the expected exception, by checking the text.